### DISCUSSION TITLE
Error on try use AppendExecutionProvider_Nnapi();

### DISCUSSION THREAD

**Discussion by sergiofreitasbra** [microsoft/onnxruntime#22956] [API Q&A]
Hi im are  get this error on use yolov11   - {Microsoft.ML.On
nxRuntime.OnnxRuntimeException: [ErrorCode:Fail] op_builder_helpers.cc:144 AddNnapiSplit count [0] does not evenly divide dimension 3 [32]  somebody know what is the isse?

**Comment by skottmckay** [comment]
NNAPI Split doesn't support as many usages as ONNX. The model must have a Split node that does an uneven split. 

NNAPI is deprecated as well, so this will never change. FWIW it's pretty random as to whether NNAPI helps or not (very device specific) so you'll get much more consistent performance with the CPU EP.

**Reply by sergiofreitasbra** [reply]
Hi thank you
